plugins/rest/aws.go

```diff
@@ -101,6 +111,101 @@ func (cs *awsEnvironmentCredentialService) credentials() (awsCredentials, error)
 	return creds, nil
 }
 
+// awsProfileCredentialService represents a credential provider for AWS that extracts credentials from the AWS
+// credentials file
+type awsProfileCredentialService struct {
```

Comment:
Interesting how the presence of comments makes `go fmt` ignore formatting of the struct. Did not know that from before :)

Reply:
How so? I think it always only aligned blocks, like
```go
type x struct {
	a    string
	bbbb bool

	cccccccc int
	d        struct{}
}

We've got four blocks (one field each) here, so it looks a little wild 😄 
```